files/scripts/swss.sh

```diff
@@ -78,11 +79,24 @@ function clean_up_tables()
     end" 0
 }
 
-startPeerService() {
+startPeerAndDependentService() {
```

Comment:
Suggest to also make the function names plural (`startPeerAndDependentServices`, `stopPeerAndDependentServices`)

Reply:
Done. Also renamed the function names to match other function names style.